circuit/src/main.nr

```diff
@@ -1,4 +1,5 @@
 mod get_account;
+
 use crate::get_account::{get_account, Account};
 
 fn main(block_no: pub Field, address: pub str<42>) -> pub Account {
```

Comment:
We should probably remove `assert(account.nonce == 0); // Temporary test` now, so we can test if get_account is working.

Reply:
Removed this assert. Now test is failing due to invalid proof